common/src/main/java/com/microsoft/identity/common/internal/controllers/ApiDispatcher.java

```diff
@@ -50,6 +52,117 @@
     private static final Object sLock = new Object();
     private static InteractiveTokenCommand sCommand = null;
 
+    public static void getAccounts(final LoadAccountCommand command) {
+        final String methodName = ":getAccounts";
+        Logger.verbose(
+                TAG + methodName,
+                "Beginning load accounts."
+        );
+        sSilentExecutor.execute(new Runnable() {
+            @Override
+            public void run() {
+                initializeDiagnosticContext();
+
+                List<AccountRecord> result = null;
+                BaseException baseException = null;
+                Handler handler = new Handler(Looper.getMainLooper());
+
+                try {
+                    //Try executing request
+                    result = command.execute();
+
+                } catch (final Exception e) {
+                    //Capture any resulting exception and map to MsalException type
+                    Logger.errorPII(
+                            TAG + methodName,
+                            "Silent request failed with Exception",
+                            e
+                    );
+                    if (e instanceof BaseException) {
```

Comment:
I believe `ExceptionAdapter` does something sane if you give it a `BaseException` -- this code probably no needed